docs/sources/oncall-api-reference/personal_notification_rules.md

```diff
@@ -35,7 +35,7 @@ The above command returns JSON structured in the following way:
 | ----------- | :------: | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `user_id`   |   Yes    | User ID                                                                                                                                                                                                                                                                                             |
 | `position`  | Optional | Personal notification rules execute one after another starting from `position=0`. `Position=-1` will put the escalation policy to the end of the list. A new escalation policy created with a position of an existing escalation policy will move the old one (and all following) down on the list. |
-| `type`      |   Yes    | One of: `wait`, `notify_by_slack`, `notify_by_sms`, `notify_by_phone_call`, `notify_by_telegram`, `notify_by_email`.                                                                                                                                                                                |
+| `type`      |   Yes    | One of: `wait`, `notify_by_slack`, `notify_by_sms`, `notify_by_phone_call`, `notify_by_telegram`, `notify_by_email`, `notify_by_mobile_app`, `notify_by_mobile_app_critical`.                                                                                                                                                                                |
```

Comment:
these two `type`s are supported by the public API, we were just missing documentation on this. Example:
```bash
data.grafana_oncall_user.joey: Reading...
data.grafana_oncall_user.joey: Read complete after 0s [id=UVNCKC46JP91X]
grafana_oncall_user_notification_rule.joey_important_step_1: Refreshing state... [id=N9SQ2K7KAK4BJ]
grafana_oncall_user_notification_rule.joey_step_1: Refreshing state... [id=NMH6B8X6CXJQK]
grafana_oncall_user_notification_rule.joey_important_step_2: Refreshing state... [id=NRVU93Z2HD4RD]
grafana_oncall_user_notification_rule.joey_step_2: Refreshing state... [id=NEM2T1S65KTSG]

Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # grafana_oncall_user_notification_rule.joey_important_step_1 will be updated in-place
  ~ resource "grafana_oncall_user_notification_rule" "joey_important_step_1" {
        id        = "N9SQ2K7KAK4BJ"
      ~ type      = "notify_by_mobile_app" -> "notify_by_mobile_app_critical"
        # (4 unchanged attributes hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.
grafana_oncall_user_notification_rule.joey_important_step_1: Modifying... [id=N9SQ2K7KAK4BJ]
grafana_oncall_user_notification_rule.joey_important_step_1: Modifications complete after 0s [id=N9SQ2K7KAK4BJ]

Apply complete! Resources: 0 added, 1 changed, 0 destroyed.
```